tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -115,3 +134,20 @@ def test_multiprocess(storage_url: str) -> None:
     assert len(trials) == n_workers * 20
 
     _check_trials(trials)
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_multiprocess_with_progbar(storage_url: str, n_jobs: int) -> None:
+    n_workers = 8
+    study_name = _STUDY_NAME
+    optuna.create_study(storage=storage_url, study_name=study_name)
+    with Pool(n_workers) as pool:
+        progress_bars = pool.map(
+            get_completed_progbar, [(study_name, storage_url, n_jobs)] * n_workers
+        )
+
+        n_completed_progbars = 0
+        for progress_bar in progress_bars:
+            if "20/20" in progress_bar and "100%" in progress_bar:
+                n_completed_progbars += 1
+        assert n_completed_progbars == n_workers
```

Comment:
Can we remove this test case?

1. `tests/storages_tests/rdb_tests/test_with_server.py` has the test cases to check the behavior of `RDBStorage` with multiple RDB servers such as MySQL and PostgreSQL. I don't think the progress bar feature depends on them.

2. The suggested test code executes `study.optimize` in parallel processes,  so each progress bar does not interact with others. So, I think we can simplify this test case by just checking the progress bar in a process.

Reply:
Ok, removed.

Reply:
Thanks!